client/my-sites/pages/page/index.js

```diff
@@ -203,6 +205,24 @@ class Page extends Component {
 		);
 	}
 
+	// Inserting new Make Homepage item to popover menu.
+	getHomepageItem() {
+		if ( this.props.hasStaticFrontPage && this.props.isPostsPage ) {
+			return null;
+		}
```

Comment:
As previously noted, using a feature flag here will allow us to split this work across multiple PRs:

```js
if ( ! isEnabled( 'manage/pages/set-front-page' ) ) {
    return null;
}
```

Reply:
What is the best way for me to add this feature flag? In my local branch, and then push that change and make a new PR? Thanks for reviewing @nylen!

Reply:
You should push all of the changes we mentioned above to this same PR. If you add new commits to your `try/add-menu-item` branch, then push, they will show up here.

Reply:
I need a little guidance for creating this feature flag as I've never done this before. 
1. Should I go into my branch's config/development.json file and add "pages/set-front-page": true ? 

2. Then, to getHomepageItem() { } I should add: 

if ( ! isEnabled( 'manage/pages/set-front-page' ) ) {
    return null;
}

^ Should I add this snippet in addition to what I already have in my getHomepageItem() function, or completely replace what's currently inside of my getHomepageItem() with the above code? Thank you!

Reply:
Hi - to explain a little bit more about the feature flag, the goal is that we developers see a button for the new "Set as front page" feature, but since it's not functional yet, we don't want our users to see it.

So we accomplish that as follows.  (1) yes, you can set the flag to true in `development.json`.  Let's call it `manage/pages/set-front-page` though - and put it right after the existing `manage/pages` flag.

Then, add the `isEnabled` check in addition to the code you already have in `getFrontPageItem`.

Reply:
I've made these changes, but when I go to push them to my branch I'm getting this error:
![error](https://user-images.githubusercontent.com/19803726/38782493-09636bfc-40ba-11e8-906e-3cad05a1e32c.png)

Is it ok that the isEnabled check is undefined right now? Should I bypass?